azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -181,12 +182,15 @@ class PipelineRootStage(PipelineStage):
     :type on_disconnected_handler: Function
     """
 
-    def __init__(self):
+    def __init__(self, pipeline_configurations=None):
         super(PipelineRootStage, self).__init__()
         self.on_pipeline_event_handler = None
         self.on_connected_handler = None
         self.on_disconnected_handler = None
         self.connected = False
+        if issubclass(pipeline_configurations.__class__, PipelineConfig):
```

Comment:
>issubclass [](start = 11, length = 10)

The more typical way is `if isinstance(pipeline_configuration, PipelineConfig):`